cdhweb/settings.py

```diff
@@ -323,6 +323,7 @@
     "fullurl",
     "django_cas_ng",
     "pucas",
+    "django_dbml",
```

Comment:
When you switch this to a dev requirement, you can't include it here — there should be some examples at the end of the settings file where dependencies are imported and added to installed apps when present.